### ISSUE TITLE
Is it possible to have two Client instances that use different tokens?

### ISSUE THREAD

**Issue by yob** [phrase/phraseapp-ruby#4]
The documented authentication technique appears to set a global token for the current process:

```ruby
auth_handler = PhraseApp::Auth::AuthHandler.new(token: "YOUR_ACCESS_TOKEN")
PhraseApp::Auth.register_auth_handler(auth_handler)
puts Client.projects_list(1,10)
```

I'd like to maintain two (or more) `Client` instances that use different user's tokens so actions can be performed as the appropriate user.

Is that possible? Something like this would be ideal:

```ruby
user1_handler = PhraseApp::Auth::AuthHandler.new(token: "USER1_ACCESS_TOKEN")
user2_handler = PhraseApp::Auth::AuthHandler.new(token: "USER2_ACCESS_TOKEN")
client1 = PhraseApp::Client.new(user1_handler)
client2 = PhraseApp::Client.new(user2_handler)
puts client1.projects_list(1,10)
puts client2.projects_list(1,10)
```

**Comment by fredv** [comment]
@yob we have a fix on the way and will release it as 1.1.0. The client will then be instantiable with Credentials. All calls will be made through the instance of the client, rather then the Client Class. This will be a breaking change. Multiple clients with different credentials are possible. I expect the release of this to happen today or tomorrow.

**Comment by yob** [comment]
That sounds great, thanks!

**Comment by kennell** [comment]
Hi @yob, we just released version 1.1.0. As @fredv pointed out, you are now able to setup two Clients with different auth tokens. Example:

```ruby
credentialsA = PhraseApp::Auth::Credentials.new(token: "ACCESS_TOKEN_A")
clientA = PhraseApp::Client.new(credentialsA)

credentialsB = PhraseApp::Auth::Credentials.new(token: "ACCESS_TOKEN_B")
clientB = PhraseApp::Client.new(credentialsB)
```
